docarray/array/array/io.py

```diff
@@ -302,6 +306,75 @@ def to_json(self) -> str:
         """
         return json.dumps([doc.json() for doc in self])
 
+    @classmethod
+    def from_files(
+        cls,
+        patterns: Union[str, List[str]],
+        recursive: bool = True,
+        size: Optional[int] = None,
+        sampling_rate: Optional[float] = None,
+        read_mode: Optional[str] = None,
+        exclude_regex: Optional[str] = None,
+        *args,
+        **kwargs,
+    ) -> 'DocumentArray':
```

Comment:
can u use TypeVar to higlight than the return document will have the same type as `cls` ?

Usually it is does by doing:

```
@classmethod
def my_method(cls: Type[T]) -> T:
   ...
```